src/GoodsList/GoodsList.tsx

```diff
@@ -0,0 +1,141 @@
+import React from 'react';
+import './GoodsList.scss';
+
+type Props = {
+  goodsFromServer: string[],
+};
+
+type SortStyle = '' | 'name' | 'length';
```

Comment:
```suggestion
type SortStyle = 'none' | 'name' | 'length';
```
Можна було б додати якесь значення, щоб виглядало змістовніше

Reply:
fixed